app/javascript/app/document-capture/image_upload.js

```diff
@@ -0,0 +1,34 @@
+import useCSRF from './hooks/use-csrf';
+
+const imageUploadURI = '/api/verify/upload';
+
+function uploadImages(frontImage, backImage, selfieImage) {
+  const csrfToken = useCSRF();
```

Comment:
The "use-" prefix here signals to me that this is intended to be used as a React hook.

>Its name should always start with use so that you can tell at a glance that the [rules of Hooks](https://reactjs.org/docs/hooks-rules.html) apply to it.

https://reactjs.org/docs/hooks-custom.html

But as used here, `uploadImages` isn't a component, so if the underlying `useCSRF` were changed to something which depended on other React hooks, it'd likely break.

Instead, I think this is something where our upload function could expect to receive the credentials as an argument (separate from payload values), and then the component which triggers that upload could choose to pass the value as provided through a React hook.

Reply:
> The "use-" prefix here signals to me that this is intended to be used as a React hook.
>
> [...]
>
> Instead, I think this is something where our upload function could expect to receive the credentials as an argument (separate from payload values), and then the component which triggers that upload could choose to pass the value as provided through a React hook.

Made CSRF part of existing `UploadContext` in ad271a0e9b036c591bde9f272a641ba2d1101f0b.